scss/index.scss

```diff
@@ -2,16 +2,17 @@
   display: flex;
   flex-flow: row wrap;
   justify-content: center;
-  width: 1000px;
+  padding: 75px 200px;
 }
 
 .list-item {
   box-shadow: 4px 4px 10px 0 $black-lighter-5;
```

Comment:
the shadow is unlikely that be the same. I believe that is a little different. 🤔